docs/UserGuide.md

```diff
@@ -70,43 +73,45 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+**Format**: `help`
 
 
 ### Adding a student: `add`
 
 Adds a student to the address book.
 
-**Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID] [owed/OWED]`
+**Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
 
 **Example:**
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography r/100 paid/100 owed/0`
 
 **Output:**
 ![addResult.jpeg](images/addResult.jpeg)
 
-<box type="important" header="#### Constraints">
+<box type="important">
 
-1. **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
-2. **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
-3. **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
-4. **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
-5. **RATE**, **PAID** and **OWED** must be at least 0 with at most 2 decimal places.
-      <i>Example: </i> `12.00`, `0.0` or `7`
-6. **SUBJECT** should only be:
+##### Constraints
```

Comment:
Could consider using same format as line 111.

Reply:
Hmmm, this is because I want to refer to it in the `edit` command